okapi-core/src/main/java/org/folio/okapi/managers/ProxyService.java

```diff
@@ -685,7 +685,7 @@ private void proxyRequestResponse10(Iterator<ModuleInstance> it,
 
   private void copyHeaders(HttpClientRequest cReq, RoutingContext ctx, ModuleInstance mi) {
     int sz = 0;
-    int limit = 5; // all headers dumped
+    int limit = 2000; // all headers dumped
```

Comment:
Can you also change

    if (sz > limit) {

to

    if (sz > limit && logger.isInfoEnabled()) {

and remove the unused StringBuilder from dumpHeaders?

Reply:
Yep. Will do